src/NServiceBus.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -174,74 +174,60 @@ void Connection_ConnectionShutdown(object sender, ShutdownEventArgs e)
 
                 if (rejectMessage)
                 {
-                    await RejectMessage(channel, message.DeliveryTag, messageId).ConfigureAwait(false);
+                    await RejectMessage(channel, message.DeliveryTag).ConfigureAwait(false);
                 }
                 else
                 {
-                    await AcknowledgeMessage(channel, message.DeliveryTag, messageId).ConfigureAwait(false);
+                    await AcknowledgeMessage(channel, message.DeliveryTag).ConfigureAwait(false);
                 }
             }
+            catch (AlreadyClosedException ex)
+            {
+                Logger.Warn($"Attempt to acknowledge/reject message {messageId} failed because the channel was closed. The message will be requeued.", ex);
+            }
             catch (Exception ex)
             {
-                Logger.Warn($"Error while attempting to process message {messageId}. The message will be rejected.", ex);
+                Logger.Warn($"Error while attempting to process message {messageId}. The message will be requeued.", ex);
```

Comment:
Not really, given the restructuring of the methods and catch blocks. Previously if you got to that catch, something really weird would have happened, so there's a good chance your connection is already lost (which automatically requeues the message), The `RejectMessage` call used to have separate logging for that case, but it doesn't now. I changed the message to indicate that the message would be requeued because A) we call `BasicReject` and it succeeds, or B) the connection is already closed and requeued anyway.